welcome/install-d-locally.md

```diff
@@ -29,17 +29,17 @@ Für viele Distributionen werden auch Pakete angeboten:
 * [Gentoo](https://wiki.gentoo.org/wiki/Dlang)
 * [OpenSuse](http://dlang.org/download.html#dmd)
 
-### Andere Kompiler
+### Andere Compiler
 
-Neben dem DMD-Referenzkompiler, der sein eigenes Backend verwendet, gibt es noch
-zwei weitere Kompiler, die über den Downloadbereich auf [dlang.org](https://dlang.org/download.html)
+Neben dem Referenz-Compiler DMD, der sein eigenes Backend verwendet, gibt es noch
+zwei weitere Compiler, die über den Downloadbereich auf [dlang.org](https://dlang.org/download.html)
 heruntergeladen werden können:
 
-* [**GDC**](http://gdcproject.org/downloads) - basierend auf dem GCC-Backend
-* [**LDC**](https://github.com/ldc-developers/ldc#installation) - basierend auf dem LLVM-Backend
+* [GDC](http://gdcproject.org/downloads) - basierend auf dem GCC-Backend
+* [LDC](https://github.com/ldc-developers/ldc#installation) - basierend auf dem LLVM-Backend
```

Comment:
In der englischen Version sind die Links fett gedruckt, am besten beibehalten und nicht rausnehmen oder alternativ auch im englischen (und damit in allen anderen Übersetzungen) ändern.